### PR TITLE
Update select.md

### PR DESCRIPTION
**Pull request by arjenpdevries**
This could be handy for people like me wanting to create a table with rowid as identifier (yes, against the recommendation :-))

### CODE REVIEW

File: docs/sql/statements/select.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ D SELECT rowid, id, content FROM t;
 In the current storage, these identifiers are contiguous unsigned integers (0, 1, ...) if no rows were deleted. Deletions introduce gaps in the rowids which may be reclaimed later. Therefore, it is strongly recommended *not to use rowids as identifiers*.
 
 > The `rowid` values are stable within a transaction.
+> The
```
Comment (Mytherin) [inline docs/sql/statements/select.md:102]: I suspect that a part of this sentence is missing

Comment (arjenpdevries) [reply docs/sql/statements/select.md:102]: Hmmm, yes... sorry!

File: docs/sql/statements/select.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ D SELECT rowid, id, content FROM t;
 In the current storage, these identifiers are contiguous unsigned integers (0, 1, ...) if no rows were deleted. Deletions introduce gaps in the rowids which may be reclaimed later. Therefore, it is strongly recommended *not to use rowids as identifiers*.
 
 > The `rowid` values are stable within a transaction.
+> The `rowid` type is
```
Comment (hawkfish) [inline docs/sql/statements/select.md:102]: I think something got lost here?

Comment (arjenpdevries) [reply docs/sql/statements/select.md:102]: Hmpf. Let me see what happened.

**Pull request merged**